src/libsyncengine/reconciliation/platform_inconsistency_checker/platforminconsistencycheckerworker.cpp

```diff
@@ -132,9 +136,10 @@ void PlatformInconsistencyCheckerWorker::blacklistNode(const std::shared_ptr<Nod
         default:
             break;
     }
-    LOGW_SYNCPAL_INFO(_logger, L"Blacklisting remote item '" << Utility::formatSyncPath(relativePath).c_str() << L"' because " << causeStr.c_str() << L".");
+    LOGW_SYNCPAL_INFO(_logger, L"Blacklisting remote item with " << Utility::formatSyncPath(relativePath).c_str() << L" because " << causeStr.c_str() << L".");
```

Comment:
With current change a log would look like `Blacklisting remote item with [path]`
If this change is to indicate both the remote and local items were blacklisted, it will be clearer to just remove the `remote` part

```suggestion
    LOGW_SYNCPAL_INFO(_logger, L"Blacklisting item " << Utility::formatSyncPath(relativePath).c_str() << L" because " << causeStr.c_str() << L".");
```

Reply:
Most of the time, at least on first sync, only remote items will be blacklisted because items are not yet synchronized.
The only case where we blacklist both is when a rename operation generated a name clash on 2 synchronized items.